src/main/java/cloud/fogbow/common/constants/AzureConstants.java

```diff
@@ -2,6 +2,12 @@
 
 public class AzureConstants {
 
+    // azure resources id
+    public static final String VIRTUAL_MACHINE_STRUCTURE = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Compute/virtualMachines/%s";
+    public static final String VIRTUAL_NETWORK_STRUCTURE = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/virtualNetworks/%s";
+    public static final String NETWORK_INTERFACE_STRUCTURE = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/networkInterfaces/%s";
+    public static final String BIGGER_STRUCTURE = NETWORK_INTERFACE_STRUCTURE;
```

Comment:
These constants have already been migrated to the AzureResourceIdBuilder class. Do you think it's more relevant that they remain here?